be/src/util/cpu_info.cpp

```diff
@@ -81,6 +81,7 @@ int CpuInfo::max_num_numa_nodes_;
 unique_ptr<int[]> CpuInfo::core_to_numa_node_;
 vector<vector<int>> CpuInfo::numa_node_to_cores_;
 vector<int> CpuInfo::numa_node_core_idx_;
+pthread_mutex_t CpuInfo::init_mutex_;
```

Comment:
1. we don't need this lock, because CpuInfo is only initialized in main function.
2. even if we need a lock, std::mutex is preferred.
3. you don't call pthread_mutex_init() to initialize this variable, it will lead process crash.

Reply:
This problem has been fixed in [1826](https://github.com/apache/incubator-doris/pull/1826) with same way, in which use a mutex to lock when init(). So I close this now.